apps/site/components/Common/Tooltip/index.module.css

```diff
@@ -0,0 +1,41 @@
+.content {
+  @apply rounded-md
+    border
+    bg-white
+    text-sm
+    font-medium
+    text-neutral-900
+    shadow-lg
+    dark:bg-neutral-950
+    dark:text-white;
```

Comment:
add `m-1.5` to avoid the sticky edge of the mobile page.
<img width="686" alt="Capture d’écran 2024-10-26 à 11 30 54" src="https://github.com/user-attachments/assets/07834db4-b600-4416-ad5d-6de46116f264">